BrainCloudClient/tests/TestEvent.cs

```diff
@@ -89,6 +89,53 @@ public void TestDeleteIncomingEvent()
 
             tr.Run();
         }
+        
+        [Test]
+        public void TestDeleteIncomingEvents()
+        {
+            TestResult tr = new TestResult(_bc);
+            
+            string eventId = SendDefaultMessage();
+            string[] eventIds = {eventId};
+            
+            _bc.EventService.DeleteIncomingEvents(
+                eventIds,
+                tr.ApiSuccess, tr.ApiError);
+
+            tr.Run();
+        }
+        
+        [Test]
+        public void DeleteIncomingEventsOlderThan()
+        {
+            TestResult tr = new TestResult(_bc);
+
+            DateTime now = DateTime.UtcNow;
+            int dateInMillis = new DateTimeOffset(now).Millisecond;
+            
+            _bc.EventService.DeleteIncomingEventsOlderThan(
+                dateInMillis,
+                tr.ApiSuccess, tr.ApiError);
+
+            tr.Run();
+        }
+        
+        [Test]
+        public void DeleteIncomingEventsByTypeOlderThan()
+        {
+            TestResult tr = new TestResult(_bc);
+
+            string eventId = SendDefaultMessage();
+            DateTime now = DateTime.UtcNow;
+            int dateInMillis = new DateTimeOffset(now).Millisecond;
+            
+            _bc.EventService.DeleteIncomingEventsByTypeOlderThan(
+                eventId,
```

Comment:
This is wrong. you need the event type, not the event id. `_eventType`

Reply:
I've made this change with the latest push.